### PR TITLE
Screencast about Docker tooling

### PR DESCRIPTION
**Pull request by xcoulon**

### CODE REVIEW

File: blog/docker_tooling_eclipse_mars.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
+= Screencast of Docker Tooling for Eclipse
+:page-date: 2015-07-01
```
Comment (maxandersen) [inline blog/docker_tooling_eclipse_mars.adoc]: I louve this! :)

File: blog/docker_tooling_eclipse_mars.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+:page-layout: blog
+:page-author: xcoulon
+:page-tags: [docker, wildfly, jbosscentral]
+
+Last week at http://devnation.org[DevNation] I had the chance to show some of the cool features that we
+provide in JBoss Developer Studio, including the Docker tooling that we've built for Eclipse Mars
+and that was released last weekfootnote:[Docker tooling is part of Developer Studio since version 9.0.0.Beta1 and JBoss Tools users can install it from https://marketplace.eclipse.org/content/jboss-tools-mars[Eclipse Marketplace] or from http://download.jboss.org/jbosstools/mars/development/updates/[our Update site].].
```
Comment (maxandersen) [inline blog/docker_tooling_eclipse_mars.adoc]: .]. should just be ].

File: blog/docker_tooling_eclipse_mars.adoc [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+I also recorded a screencast that highlights some of those Docker tooling features.
+The video shows how to:
+
+ * Connect to a local Docker engine (0:32)
```
Comment (maxandersen) [inline blog/docker_tooling_eclipse_mars.adoc]: since these are youtubes - just add #t=0m32s

File: blog/docker_tooling_eclipse_mars.adoc [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ * Start a PostgreSQL container with a custom username and password (1:52)
+ * Build a custom image for Wildfly to include the PostgreSQL JDBC driver and configure a datasource (3:20)
+ * Start a WildFly container linked to PostgreSQL, with exposed ports and a volume mounted on the host machine (6:44)
+ * Import a Quickstart project using JBoss Central (8:30)
+ * Deploy the project on the WildFly container using a Server Adapter with custom settings (9:55)
+ * Connect to the JVM using the Remote Debugging tools (13:25)
+
+video::rzh-95FA6NA[youtube]
```
Comment (maxandersen) [inline blog/docker_tooling_eclipse_mars.adoc]: haven't seen the full video yet but what I saw was great.

Main problem was that the video is only available in 480p so its quite blurry.
we should get this added to /video page too.

File: blog/docker_tooling_eclipse_mars.adoc [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+
+I commented all the tasks as I recorded this screencast to highlight the important steps, but you'll have to cope with my accent ;-)
+Beware that if you run 'boot2docker', you'll need to use a specific IP address instead of 'localhost' when creating the Server adapter in Eclipse (trust me, I made this mistake before...)
```
Comment (maxandersen) [inline blog/docker_tooling_eclipse_mars.adoc]: this is not exactly true.

The only time localhost would work is when running on linux with native docker - anywhere else dockerhost ip/name is what is needed. Which is why I always recommend to setup alias in /etc/host to map dockerhost  to the ip no matter what OS you are on.

File: stylesheets/jbt-styles.scss [hunk 6]
```diff
@@ -403,41 +417,41 @@ kbd {
...
-  
+
   iframe {
     margin-left: auto;
     margin-right: auto;
     display: block;
   }
-  
+
   @media (min-width: 981px) {
```
Comment (maxandersen) [inline stylesheets/jbt-styles.scss]: was this intentional change ?

File: blog/docker_tooling_eclipse_mars.adoc (hunk 5 above)
Comment (xcoulon) [inline blog/docker_tooling_eclipse_mars.adoc]: huh ? not following you here: I'm specificaly mentioning the case of boot2docker. Granted, having 'dockerhost' is a workaround.

File: stylesheets/jbt-styles.scss (hunk 6 above)
Comment (xcoulon) [inline stylesheets/jbt-styles.scss]: all those changes occurred when I edited this file in Atom. I can revert those changes if needed

File: blog/docker_tooling_eclipse_mars.adoc (hunk 1 above)
Comment (xcoulon) [inline blog/docker_tooling_eclipse_mars.adoc]: oh really ? ;-)

File: blog/docker_tooling_eclipse_mars.adoc (hunk 5 above)
Comment (maxandersen) [inline blog/docker_tooling_eclipse_mars.adoc]: you say the issue is when you run boot2docker - I'm just saying this is not unique to boot2docker. You will *always* have to use dockerhost ip and it is only same as localhost if you run on Linux with native docker.

Comment (xcoulon) [inline blog/docker_tooling_eclipse_mars.adoc]: ok, got it

**Pull request merged**